nautobot/docs/models/extras/dynamicgroup.md

```diff
@@ -1,7 +1,166 @@
 # Dynamic Groups
 
-A Dynamic Group provides a way to organize objects of the same Content Type by matching filters. The Dynamic Group can be used to create unique groups of objects matching a given filter, such as Devices for a specific site location or set of locations. As indicated by the name, Dynamic Groups update in real time as objects are created, updated, or deleted.
+A Dynamic Group provides a way to organize objects of the same Content Type by matching filters. The Dynamic Group can be used to create unique groups of objects matching a given filter, such as Devices for a specific site location or set of locations. As indicated by the name, Dynamic Groups update in real time as member objects are created, updated, or deleted.
 
-When creating a Dynamic Group, one must select a Content Type to which it is associated, for example `dcim.device`. The filtering parameters saved to the group behave as a bi-directional search query that used to identify members of that group, and can also be used to determine from an individual object in which Dynamic Groups it is a member.
+When creating a Dynamic Group, one must select a Content Type to which it is associated, for example `dcim.device`. The filtering parameters saved to the group behave as a bi-directional search query that used to identify members of that group, and can also be used to determine from an individual object the list of Dynamic Groups to which it belongs.
 
 Once created the Content Type for a Dynamic Group may not be modified as this relationship is tightly-coupled to the available filtering parameters. All other fields may be updated at any time.
+
+## Basic Filtering
+
+Dynamic Groups filtering is powered by **FilterSet** objects underneath the hood. Basic filtering is performed using the `filter` that is defined on a given Dynamic Group.
+
+An object is considered to be a member of a Dynamic Group if it is of the same Content Type and it is not excluded by way of any of the filter critera specified for that group. By default, if a group has an empty filter (`{}`) it will include all objects of the matching Content Type, just as a defaut list view of objects would prior to any filter fields being filled in the web UI.
+
+For example, for a Dynamic Group with Content Type of `dcim.device` and an empty filter, the list of members would be equivalent to the queryset for `Device.objects.all()`.
+
+!!! warning
+    This behavior was changed in v1.4.0. In v1.3.0 the default for a group with an empty filter was to fail "closed" and have zero members. As of v1.4.0, this behavior has been inverted to include all objects matching the content type by default instead of matching no objects. This was necessary to implement the progressive layering of child filters similarly to how we use filters to reduce desired objects from basic list view filters. This will described in more detail below.
```

Comment:
```suggestion
    This behavior was changed in v1.4.0. In v1.3.x the default for a group with an empty filter was to fail "closed" and have zero members. As of v1.4.0, this behavior has been inverted to include all objects matching the content type by default instead of matching no objects. This was necessary to implement the progressive layering of child filters similarly to how we use filters to reduce desired objects from basic list view filters. This is described in more detail below.
```